auth4genai/how-tos/analyze-strava-activities.mdx

```diff
@@ -20,7 +20,7 @@ Before using this example, make sure you:
 ## 1. Define the Vercel AI Tool and backend API Route
 
 Define a `getActivities` tool that uses GPT-4 to fetch and analyze a user's recent fitness activity from Strava:
-- When the tool calls `getAccessTokenForConnection()` to fetch a Spotify access token, pass in `strava-custom` as the connection name.
+- When the tool calls `getAccessTokenFromTokenVault()` to fetch a Spotify access token, pass in `strava-custom` as the connection name.
```

Comment:
There is still a reference to `getAccessTokenForConnection()` in the code block below